capstone-frontend/src/App.jsx

```diff
@@ -37,8 +24,6 @@ class App extends Component {
 
   fetchBookLists = async () => {
 
-    // TODO(#74): Add Spinner to LeftSide when fetching BookList from App.jsx
```

Comment:
Why did you remove this TODO?

Reply:
I tried to added in this PR. But it turn out it will be very hard and complex to add it. I don't think its worth spending the time trying to implement if its just a minor feature